setup.py

```diff
@@ -247,7 +247,9 @@ def copy_update(dir1, dir2):
         args = ''
         if CC is not None:
             args += ' CC="%s"' % ' '.join(CC[:1])
-            args += ' CFLAGS="%s"' % ' '.join(CC[1:])
+            args += ' CFLAGS="%s -fvisibility=hidden"' % ' '.join(CC[1:])
```

Comment:
Is this needed?  Was using the .o files good enough?

Reply:
We didn't test that with Klaus. I will rebuild and check the libs locally. 

However given the docs on gcc, I think we want both.

Reply:
Actually I think we need it. Before I added the static qualifier to some helper functions in the extension C code, those were appearing as exported symbols in the final lib. I don't think the entire cfitsio code base has static applied.

Reply:
Yup. The symbols are visible without those flags. We did not test explicitly if the thing you are asking about.